### PR TITLE
Clarify variable names, fix clippy nightly.

### PR DESCRIPTION
**Pull request by feuerste**

### CODE REVIEW

File: src/octree/generation.rs [hunk 1]
```diff
@@ -159,7 +159,7 @@ fn split_node<'a, P>(
     scope: &Scope<'a>,
     octree_data_provider: &'a OnDiskDataProvider,
     octree_meta: &'a octree::OctreeMeta,
-    query_attributes: &'a HashMap<String, AttributeDataType>,
+    attribute_data_types: &'a HashMap<String, AttributeDataType>,
```
Comment (catevita) [inline src/octree/generation.rs]: nit: in point_cloud/generation this very Hashmap is called just attributes or attributes_with_xyz

Comment (feuerste) [reply src/octree/generation.rs]: Yes, then we should replace it there as well, but this is another repo...

Comment (feuerste) [reply src/octree/generation.rs]: Actually here we really have attributes without xyz.

**Comment by feuerste** [comment]
Thanks for your review!

**Pull request merged**